stdlib/camlinternalAtomic.mli

```diff
@@ -0,0 +1,29 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                         The OCaml programmers                          *)
```

Comment:
Here I would reuse the copyright header of `stdlib__atomic.mli` (which credits Stephen iirc.), because this is really the same interface.

Reply:
Done (and to be really strict, I add mine since I wrote the comment, so as to take the blame of the mess).